configurations/morcore2esuites/Configuration_TaakApiInt.xml

```diff
@@ -0,0 +1,161 @@
+            <Forward name="success" path="JuisteFormat"/>
+            <Forward name="Exception" path="failure"/>
+        </JsonPipe>
+
+        <XsltPipe name="JuisteFormat" styleSheetName="xsl/TaakApiInt/GetNotification.xsl" outputType="XML">
+            <Forward name="success" path="XmlCheck"/>
+            <Forward name="Exception" path="failure"/>
+        </XsltPipe>
+
+        <XmlValidatorPipe name="XmlCheck" schema="xsd/TaakApiInt/ValidateGetMelding.xsd" root="root">
+            <Forward name="success" path="Xml2Json"/>
+            <Forward name="Exception" path="failure"/>
+        </XmlValidatorPipe>
+
+        <JsonPipe name="Xml2Json" direction="XML2JSON">
+            <Forward name="success" path="Exit"/>
+            <Forward name="Exception" path="failure"/>
+        </JsonPipe>
+
+        <!-- Mogelijke config update mogelijk ZIE HIERONDER!
+
+        <JsonXsltPipe name="CorrectResponse" styleSheetName="xsl/TaakApiInt/GetNotification.xsl">
+            <Forward name="success" path="JsonValidator"/>
+            <Forward name="Exception" path="failure"/>
+        </JsonXsltPipe>
+
+        <Json2XmlValidatorPipe name="JsonValidator" schema="xsd/TaakApiInt/ValidateGetMelding.xsd" root="root" inputFormatSessionKey="InputFormat" outputFormat="JSON">
+            <Forward name="success" path="Exit"/>
+            <Forward name="Exception" path="failure"/>
+        </Json2XmlValidatorPipe> -->
+
+        </Pipeline>
+    </Adapter>
+
+    <Adapter name="NewResponse_NotificationTasks">
+        <Receiver name="Response_NotificationTasks">
+            <JavaListener name="Response_Notificationtasks"/>
+        </Receiver>
+
+        <Pipeline>
+
+            <Exits>
+                <Exit name="Exit" state="SUCCESS"/>
+                <Exit name="failure" state="REJECTED"/>
+            </Exits>
+
+        <SenderPipe name="Send2MorCore">
+            <HttpSender name="GetNotificationTasks" methodType="GET" url="/melding/{meldingUuid}/taakopdracht"/>
+            <Forward name="success" path="Json2Xml"/>
+        </SenderPipe>
+
+        <JsonPipe name="Json2Xml" direction="JSON2XML" prettyPrint="true">
+            <Forward name="success" path="JuisteFormat"/>
+            <Forward name="Exception" path="failure"/>
+        </JsonPipe>
+
+        <XsltPipe name="JuisteFormat" styleSheetName="xsl/TaakApiInt/GetNotificationTasks.xsl" outputType="XML">
+            <Forward name="success" path="XmlCheck"/>
+            <Forward name="Exception" path="failure"/>
+        </XsltPipe>
+
+        <XmlValidatorPipe name="XmlCheck" schema="xsd/TaakApiInt/ValidateGetMeldingTaak.xsd" root="root">
+            <Forward name="success" path="Xml2Json"/>
+            <Forward name="Exception" path="failure"/>
+        </XmlValidatorPipe>
+
+        <JsonPipe name="Xml2Json" direction="XML2JSON">
+            <Forward name="success" path="Exit"/>
+            <Forward name="Exception" path="failure"/>
```

Comment:
Change to "name="exception"". Forwards are defined by the pipe and cannot be named to what you see fit. You can view the pipe in the Frank!Doc to see what forwards a pipe has.

Additionally, set the path to an exit that ends in the ERROR state.